src/Sylius/Bundle/ApiBundle/DataTransformer/OrderItemIdAwareInputCommandDataTransformer.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\DataTransformer;
+
+use Sylius\Bundle\ApiBundle\Command\OrderItemIdAwareInterface;
+
+class OrderItemIdAwareInputCommandDataTransformer implements CommandDataTransformerInterface
+{
+    public function transform($object, string $to, array $context = [])
+    {
+        $object->setOrderItemId($this->getOrderItemIdFromUri($context['request_uri']));
+
+        return $object;
+    }
+
+    public function supportsTransformation($object): bool
+    {
+        return $object instanceof OrderItemIdAwareInterface;
+    }
+
+    private function getOrderItemIdFromUri(string $uri): string
+    {
+        $explodedUri = explode('/', $uri);
+        return end($explodedUri);
+    }
```

Comment:
To be honest, I would be for removing this method and this simple logic to the method above